tests/Fixtures/ConvertConfig/formatters.yaml

```diff
@@ -10,3 +10,14 @@ default:
             other_property: value
     extensions:
         custom_extension.php: ~
+
+with_options:
+    formatters:
+        junit:
+            output_path: build/logs/junit
+        progress:
```

Comment:
Maybe also add one of the base options (`output_path` for example) to the progress and pretty formatters to confirm that they are preserved?

Reply:
Good idea - a couple were covered already in the other profile but worth having more coverage.